.github/workflows/wdio-single-browser.yml

```diff
@@ -88,12 +88,12 @@ jobs:
             ${{ inputs.additional-flags || '' }}
       - name: Find pull request
         id: pull-request-target
-        if: ${{ inputs.coverage }}
+        if: ${{ always() && inputs.coverage }}
```

Comment:
Do we want this to run even if the action is cancelled?

Reply:
I think if you cancel a workflow it cancels all jobs and steps...right?

Reply:
Just read the warning in the GHA docs. I will change this real quick.